backend.go

```diff
@@ -11,6 +11,38 @@ import (
 	"github.com/hashicorp/vault/sdk/queue"
 )
 
+type backend struct {
```

Comment:
This backend struct appears new in the diff, but it is actually relocated. Comments below highlight new members introduced by this PR.